src/layout/Footer.jsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import { Facebook, Twitter, RssFeed } from "@material-ui/icons";
+import Grid from "@material-ui/core/Grid";
+
+/* Defines CSS classes to use for all instances of this component. This is
+  actually a custom React hook. */
+const useStyles = makeStyles((theme) => ({
+  footerLink: {
+    color: theme.palette.text.secondary,
+  },
+}));
 
 const Footer = ({ title }) => {
-  return <>&copy; {title} 2020</>;
+  const classes = useStyles(); //consumes ThemeProvider 
+  return (
+    <>
+      <hr />
+      <Typography
+        variant="h5"
+        color="textSecondary"
+        align="center"
+        noWrap
+      >
+        <Link
+          to="/"
+          component={GatsbyLink}
+          color="inherit"
+        >
+          {title}
+        </Link>
+      </Typography>
+      <Grid 
+        container 
+        direction="row"
+        justify="center" 
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item>
+          <Link
+            href="https://www.facebook.com/huntercsblog"
+          >
+            <Facebook />
+          </Link>
+        </Grid>
+        <Grid item>
+          <Twitter />
+        </Grid>
+        <Grid item>
+          <RssFeed />
+        </Grid>
+      </Grid>
+      <Grid 
+        container 
+        direction="row"
+        justify="center" 
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item>
+          <Link 
+            href="mailto:icarus@huntercs.club"
```

Comment:
Should we use a `mailto` or have a contact page with the email address as well as ways to contact 'important people' (eg managing editors of departments, editor in chiefs, etc etc)?

Reply:
You're right, it would be more clear to have a full page that lists important names along with their `mailto` emails. I'll make an issue for a contact page

Reply:
#16